shell.nix

```diff
@@ -9,16 +10,42 @@
 # Also look into direnv: https://direnv.net/, this can make it so that you can
 # automatically get your environment set up when you change folders into the
 # project.
-{ pkgs ? import <nixpkgs> {} }:
 
-pkgs.mkShell {
-  # This specifies the tools that are needed for people to get started with
-  # development. These tools include:
-  #  - The Go compiler toolchain (and all additional tooling with it)
-  #  - gotools for goimports, a robust formatting tool for Go source code
-  #  - gopls, the language server for Go to increase editor integration
-  #  - git, the version control program (used in some scripts)
-  buildInputs = with pkgs; [
-    go gotools gopls git
-  ];
-}
+{
+	pkgs ? import <nixpkgs> {},
+	nixosUnstable ? import (fetchTarball https://github.com/NixOS/nixpkgs/archive/refs/heads/master.tar.gz) { },
```

Comment:
use `nixpkgs-unstable.tar.gz` rather than `master.tar.gz`. The master branch only passes basic CI checks, whereas the release branches only get bumped after Hydra pre-builds all the packages and runs a more extensive regression suite.